examples/hybrid/sphere/held_suarez_rhoe.jl

```diff
@@ -19,6 +19,7 @@ jacobian_flags = (; ∂ᶜ𝔼ₜ∂ᶠ𝕄_mode = :no_∂ᶜp∂ᶜK, ∂ᶠ
 
 # Additional values required for driver
 upwinding_mode = :third_order
+z_stretch_scale = FT(7e3)
```

Comment:
Note to self: 
This parameter is not needed here anymore since now it is set directly in the driver.
Also, the following plot level change should be applied only when run in stretched mode, not in uniform, so need to propagate that info somehow from driver. Will think about it.